### DISCUSSION TITLE
Office 365 programs only start with "run as administrator

### DISCUSSION THREAD

**Discussion by tepollack** [ramensoftware/windhawk#207] [Q&A]
First time here and I'm not sure if I am posting in the right group.  Nonetheless.

After installing Windows update: (KB5037771), I could only open Office 365 programs by using "run as administrator".  Searching the web I discovered that Windhawk "Taskbar height and icon size" was causing the problem.  The solution was to set up an exception in Windhawk → Settings → Advanced settings → More advanced settings → Process exclusion list. Add "C:\Program Files\Microsoft Office\*" to the list.

Sorry if this is old news.

**Comment by groovyclam** [comment]
It's not just "Taskbar height and icon size" I think it is Windhawk itself, even with no mods active does this.

The process workaround you posted doesn't fix things for me.

**Reply by EvBear** [reply]
Same, I tried the workaround and it didn't work so I closed Windhawk, will just have to live without it until the issue is resolved.

**Reply by m417z** [reply]
@EvBear it should work, please try following the instructions and the video recording here: https://github.com/ramensoftware/windhawk/issues/198#issuecomment-2163986031.

**Reply by groovyclam** [reply]
@EvBear - DO NOT PUT " " QUOTE MARKS around any paths you put in the exclusion settings and it should work.
Make sure your path to Office is correct as it can vary from install to install.
My exclusions are ( but check your path to "Office16" folder is the same ):
C:\Program Files\Microsoft Office\root\Office16\WINWORD.EXE
C:\Program Files\Microsoft Office\root\Office16\EXCEL.EXE
C:\Program Files\Microsoft Office\root\Office16\POWERPNT.EXE
C:\Program Files\Microsoft Office\root\Office16\MSPUB.EXE

**Comment by tepollack** [comment]
Sorry to hear that.

**Comment by groovyclam** [comment]
Hang on - your fix does work but the quote marks in your post should NOT be entered into the exclusion list

**Comment by tepollack** [comment]
Great.  It's always the little things that screw everything up.

On Thu, Jun 20, 2024 at 12:02 PM groovyclam ***@***.***>
wrote:

> Hang on - your fix does work but the quote marks in your post should NOT
> be entered into the exclusion list
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ramensoftware/windhawk/discussions/207#discussioncomment-9830919>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BJKBQYBFPLMY7TZ6ERF3SQLZIL4KTAVCNFSM6AAAAABJSIMTXGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TQMZQHEYTS>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Tom Pollack

**Comment by m417z** [comment]
Please refer for the following issue for up-to-date information: https://github.com/ramensoftware/windhawk/issues/198.

**Comment by nandeve** [comment]
It's not just Office software, Battlefield 5 also fails to start due to Windhawk. I want to know why, and there may be more software that fails to start due to Windhawk. Whenever I encounter this kind of problem, I immediately suspect that it is a Windows problem.
